common/rfb/JpegCompressor.h

```diff
@@ -49,7 +49,7 @@ namespace rfb {
 
     inline rdr::U8* getstart() { return start; }
 
-    inline int overrun(int itemSize, int nItems) {
+    inline size_t overrun(size_t itemSize, size_t nItems) {
```

Comment:
The implicit conversion upsets clang11's default configuration and stricter gcc options.